Dockerfile.WebStore

```diff
@@ -33,14 +33,17 @@ COPY EndPointCommerce.WebStore/. ./EndPointCommerce.WebStore/
 
 # build app
 WORKDIR /source/EndPointCommerce.WebStore
-RUN dotnet publish -c release -o /app --no-restore
+RUN dotnet publish -c release -o /home/app --no-restore
 
 # final stage/image
 FROM mcr.microsoft.com/dotnet/aspnet:9.0
 
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && apt-get -y install --no-install-recommends curl \
     && rm -rf /var/lib/apt/lists/*
-WORKDIR /app
-COPY --from=build /app ./
+
+USER app
+WORKDIR /home/app
+COPY --chown=app --from=build /home/app ./
+RUN mkdir -p /home/app/data-protection-keys
```

Comment:
We should remove this and all mention or data-protection-keys for the web store. It doesn't have them anymore, because now its a purely frontend app.